### ISSUE TITLE
File "/proc/self/fd/12", line 1 404: Not Found

### ISSUE THREAD

**Issue by gustawdaniel** [Sharktheone/arch-mojo#28]
```
python --version          
Python 3.12.7
```

```
uname -a
Linux hyperbook 6.11.5-arch1-1 #1 SMP PREEMPT_DYNAMIC Tue, 22 Oct 2024 18:31:38 +0000 x86_64 GNU/Linux
```

```
python <(curl -sSL https://raw.githubusercontent.com/Sharktheone/arch-mojo/main/src/install.py)
  File "/proc/self/fd/12", line 1
    404: Not Found
    ^^^
SyntaxError: illegal target for annotation
```

**Comment by Sharktheone** [comment]
Ah, thanks for pointing that out. The `install.py` file was renamed, so this is the updated command
```
python <(curl -sSL https://raw.githubusercontent.com/Sharktheone/arch-mojo/main/src/install_libs.py)
```

**Comment by gustawdaniel** [comment]
Thanks for support. Now i have the following error

```
python <(curl -sSL https://raw.githubusercontent.com/Sharktheone/arch-mojo/main/src/install_libs.py)
Installing libraries to `/home/daniel/.local/lib/arch-mojo` for `x86_64-linux-gnu`
x - debian-binary
x - control.tar.xz
x - data.tar.xz
x - debian-binary
x - control.tar.xz
x - data.tar.xz

TL;DR: If you see errors, ignore them or report them to https://https://github.com/Sharktheone/arch-mojo and restart your shell

Please note that you might be seeing some errors about some components that weren't installed correctly

For more information see here: https://github.com/Sharktheone/arch-mojo?tab=readme-ov-file#missing-shared-libs

Please do not report any installation errors to Modular, as this is not an official installation method

If you encounter any issues, please report them to https://github.com/Sharktheone/arch-mojo/issues
It would also be nice if you starred the repo, thanks! ❤️
Please restart your shell or re-soruce your shell configuration file
```

magic is installed

```
➜  ~ magic                                      
magic - A high level package management tool by Modular for developing with Mojo and MAX.

To get started, run `magic init` in your project directory.

To see all available commands, run `magic --help` or `magic help`.

Usage: magic [OPTIONS] <COMMAND>

Commands:
  init         Creates a new project
  add          Adds dependencies to the project [aliases: a]
  remove       Removes dependencies from the project [aliases: rm]
  install      Install all dependencies [aliases: i]
  update       Update dependencies as recorded in the local lock file
  run          Runs task in project [aliases: r]
  shell        Start a shell in the pixi environment of the project [aliases: s]
  shell-hook   Print the pixi environment activation script
  project      Modify the project configuration file through the command line
  task         Interact with tasks in the project
  list         List project's packages [aliases: ls]
  tree         Show a tree of project dependencies [aliases: t]
  global       Subcommand for global package management actions [aliases: g]
  config       Configuration management
  info         Information about the system, project and environments for the current machine
  search       Search a conda package
  self-update  Update magic to the latest or a specific version
  clean        Clean the parts of your system which are touched by pixi. Defaults to cleaning the environments and task cache. Use the `cache` subcommand to clean the cache
  completion   Generates a completion script for a shell
  telemetry    Configure how telemetry data is emitted from magic and Modular packages
  help         Print this message or the help of the given subcommand(s)

Options:
  -v, --verbose...     Increase logging verbosity
  -q, --quiet...       Decrease logging verbosity
      --color <COLOR>  Whether the log needs to be colored [env: MAGIC_COLOR=] [default: auto] [possible values: always, never, auto]
      --no-progress    Hide all progress bars [env: MAGIC_NO_PROGRESS=]
  -h, --help           Print help
  -V, --version        Print version
```

but when I type

```
magic init mojo-hello-world --format mojoproject 
```

then terminal do not produce any output and freezes forever.

I was able to install on ubuntu and mac os, but on arch no. 

I found suggestion to use docker version instead https://gist.github.com/Sharktheone/79da849c96db13f21eefa2be9430d9ec but not tried yet.

**Comment by Sharktheone** [comment]
Interesting... I am not sure, if this is an issue with this script, since it only installs libraries for mojo. Since magic itself is statically linked it *should* work.
On my system it seems to work.

Can you confirm that magic is statically linked for you by running
```sh
ldd $(which magic)
```
